napi/parser/wrap.js

```diff
@@ -0,0 +1,42 @@
+module.exports.wrap = function wrap(result) {
```

Comment:
Unfortunately, Vite doesn't allow cjs here. For Vite usage, it requires `optimizeDeps.exclude: ['oxc-parser']` and that means `wasm.js` file needs to be served unbundled and it needs to be esm.

In https://github.com/oxc-project/oxc/pull/9793, I just copied this one as `wrap.mjs` with `export function wrap`.

Reply:
Coming from https://github.com/oxc-project/playground/pull/67#discussion_r1986032378, technically Vite users can make it work without `optimizeDeps.exclude`, so this may not strictly necessary.

Using cjs will break, Vitest browser mode https://github.com/oxc-project/oxc/pull/9793 since locally file cannot be optimized, but this can be probably also worked around by patching `module.export.wrap -> export ` from transform plugin just for this file.